doc/Running_Motr_Across_a_Cluster.md

```diff
@@ -101,9 +100,9 @@ This document provides information on how to build motr from source and then run
 	                 #profiles:
 	                 #  - name: default
 	                 #    pools: [ the pool ]
-                ```
+        	```
             > Whereas a CDF with 3 nodes should look like this:
-               
+		    ```yml
```

Comment:
check tab

Reply:
Actually a tab is needed there in order to mark it as code block of particular language

Reply:
Had a conversation with @prasanna46dk and understood the problem. will fix it.